### ISSUE TITLE
Problem with "Create profile to save progress"

### ISSUE THREAD

**Issue by mjp100** [MicrosoftDocs/feedback#1234]
When trying to register for the docs.microsoft.com/learn site I am presented with the "Create profile to save progress" page. Once filled in clicking on the register button does nothing. I get no errors just the button showing it is being depressed

I have tried this with a personal @outlook.com address and a work based Office365 address with the same issue. 

I have tried this on macOS Mojave (both Safari, Chrome and Firefox), Windows 10 (Edge), IOS 12 (Safari)

**Comment by mrobson1975** [comment]
I have the same issue; would appreciate posting a solution.

**Comment by s9s9s9s9** [comment]
I am having the same issue.

**Comment by simondavidhill** [comment]
I have the same issue.

**Comment by s9s9s9s9** [comment]
I selected United States in the country dropdown list and it worked :-)

**Comment by simondavidhill** [comment]
Workaround confirmed.

**Comment by mjp100** [comment]
Yes it looks like selecting the US allows the register button to work. I noticed that with US selected it does not show the accept terms and conditions check box

**Comment by stillface** [comment]
for me is not working even I selectes US. 
There is a message: "Something went wrong. Please try again later."

**Comment by mjp100** [comment]
Just tried with my personal account, selected UK (since I'm UK based) and did not tick the "I agree" button. It allowed me to register. Looks like there is a bug behind the "I agree" checkbox, and stops the register button from working. Possibly the true/false is the wrong way round in the code behind the page.

**Comment by stillface** [comment]
this not Works for me... I tried many country with and without "I agree" checkbox….
